samples/gui_test_wgpu/src/gui_test_wgpu.zig

```diff
@@ -106,7 +106,7 @@ fn create(allocator: std.mem.Allocator, window: *zglfw.Window) !*DemoState {
 
     style.window_min_size = .{ 320.0, 240.0 };
     style.window_border_size = 8.0;
-    style.scrollbar_size = 6.0;
+    style.scrollbar_size = 12.0;
```

Comment:
I'm nervous about this because I don't understand why it's changed.

Reply:
FIxed. My bad ;)

Reply:
But without this change, the scroll bar is thinner than before

Reply:
Yes because 6 is really thin, default in imgui is `12`. strange i know 😅

Reply:
It seems about right. Maybe something was fixed upstream. Let's go with it.